relayer/packet-tx.go

```diff
@@ -65,21 +65,32 @@ func (c *Chain) SendTransferMsg(ctx context.Context, log *zap.Logger, dst *Chain
 		}
 	}
 
+	clientHeight := h.GetLatestHeight().GetRevisionHeight()
+
 	switch {
 	case toHeightOffset > 0 && toTimeOffset > 0:
-		timeoutHeight = h.GetHeight().GetRevisionHeight() + toHeightOffset
+		timeoutHeight = clientHeight + toHeightOffset
 	case toHeightOffset > 0:
-		timeoutHeight = h.GetHeight().GetRevisionHeight() + toHeightOffset
+		timeoutHeight = clientHeight + toHeightOffset
 		timeoutTimestamp = 0
 	case toTimeOffset > 0:
 		timeoutHeight = 0
 	case toHeightOffset == 0 && toTimeOffset == 0:
-		timeoutHeight = h.GetHeight().GetRevisionHeight() + 1000
+		timeoutHeight = clientHeight + 1000
```

Comment:
Since you're touching this code anyways would you make this value, `1000`, a const called something like `defaultTimeoutOffset.

Reply:
good call, done